src/components/graph/graph.ts

```diff
@@ -111,17 +111,22 @@ export default class GraphComponent {
     this._c10 = d3.scaleOrdinal(d3.schemeCategory10)
 
     this._actionStream = actionStream
+    let isSingleClick: boolean = true
 
     this._svg = d3.select(host)
       .append('svg')
       .attr('width', dimensions.width)
       .attr('height', dimensions.height)
       .on('click', () => {
         d3.event.stopPropagation()
-        this._actionStream!.next(new BackgroundClickAction())
+        isSingleClick = true
+        setTimeout(() => { if (isSingleClick) {
+          this._actionStream!.next(new BackgroundClickAction())
+        } }, 300)
```

Comment:
make the 300 a static property on this class and move it to the top along with the other statics. Add a comment above the value too just to document why the value is 300.